test/change.test.js

```diff
@@ -601,10 +610,90 @@ describe('Change', function() {
           modelName: updateRecord.modelName,
           prev: null, // this is the current local revision
           rev: 'new-rev',
+          tenant: null,
         });
 
         done();
       });
     });
   });
 });
+
+describe('Change with multi tenancy', function() {
+  var ChangeWithTenant;
+  var TestModelWithTenant;
+  var tenantProperty = 'tenantId';
+
+  beforeEach(function() {
+    var memory = loopback.createDataSource({
+      connector: loopback.Memory
+    });
+    TestModelWithTenant = loopback.PersistedModel.extend('ChangeTestModel',
+      {
+        id: {id: true, type: 'string', defaultFn: 'guid'}
+      },
+      {
+        trackChanges: true,
+        tenantProperty: tenantProperty
+      });
+    this.modelName = TestModelWithTenant.modelName;
+    TestModelWithTenant.attachTo(memory);
+    ChangeWithTenant = TestModelWithTenant.getChangeModel();
+
+    TestModelWithTenant.prototype.fillChangeTenantValue = function(change, cb) {
+      var model = TestModelWithTenant;
+      var tenant = null;
+      var tenantProperty;
+      var inst = this;
+
+      if (model && model.settings && model.settings.tenantProperty) {
+        tenantProperty = model.settings.tenantProperty;
+      }
+
+      if (tenantProperty && inst && inst[tenantProperty]) {
+        tenant = inst[tenantProperty];
+      }
+
+      change.tenant = tenant;
+
+      cb(null);
+    };
+  });
+
+  describe('change.rectify - tenant variant', function() {
```

Comment:
No need to repeat the fact that our Change model has extra properties, that was already said in `describe()`.